engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/exception/FlinkEspExceptionHandler.scala

```diff
@@ -13,13 +16,22 @@ object FlinkEspExceptionHandler {
   }
 }
 
-trait FlinkEspExceptionHandler extends EspExceptionHandler {
+trait FlinkEspExceptionHandler extends Lifecycle{
```

Comment:
FlinkNuExceptionHandler

Reply:
Do we need this trait at all in API? I think the idea is to have only `ConfigurableExceptionHandler` and `ExceptionConsumer` as part of the API?

Reply:
In fact we don't need it. I'll remove it :+1:

Reply:
removal: https://github.com/TouK/nussknacker/pull/2539/commits/33b2878e90a3119f3b5f01f90a9e88517442137f
rename: https://github.com/TouK/nussknacker/pull/2539/commits/8edb4673a182a5e75b12c72386fbb128f4977868